packages/shared/src/lib/cores.ts

```diff
@@ -0,0 +1,10 @@
+import type { LoggedUser } from './user';
+import { CoresRole } from './user';
+
+export const verifyCoresRole = (
+  user: LoggedUser,
+  coresRole: CoresRole,
+): boolean => user?.coresRole === coresRole;
+
+export const checkCoresRoleNotNone = (user: LoggedUser): boolean =>
```

Comment:
maybe call it hasAccessToCores, with `Not` the `!checkCoresRoleNotNone()` sounds off, not not none?